### ISSUE TITLE
URL includes encoded documentation when drop-down value changed

### ISSUE THREAD

**Issue by lukaszgryglicki** [cncf/devstats.archive#116]
Bug: URL includes encoded documentation when drop-down value changed
When such URL is copied and then pasted it is no longer working.

**Comment by lukaszgryglicki** [comment]
Will work on this tomorrow.

**Comment by lukaszgryglicki** [comment]
I've researched Grafana configuration options and didn't find any solution for this.
I've googled a bit and found information that grafana always include all template variable values in the url.
But this template variable is hidden and cannot be changed from the UI. It just holds docs.
So I think there is no sense to include its value in URL.
I've created issue on Grafana asking to add support for skipping some template variables values in the url.
Other solution might be that Grafana should skip values of hiden template variables (user can't change them anyway).
https://github.com/grafana/grafana/issues/12174
cc @jberkus @dankohn

**Comment by lukaszgryglicki** [comment]
I will try to implement this in Grafana tomorrow or Thursday.
I'll also try to upstream this by creating PR on grafana/grafana.

**Comment by lukaszgryglicki** [comment]
Created PR that implements this: https://github.com/grafana/grafana/pull/12259
Issue here: https://github.com/grafana/grafana/issues/12174#issuecomment-396921234

I need to find a way to create package from a branch - so I can update our Grafanas to have that option turned on before it gets merged.

**Comment by lukaszgryglicki** [comment]
Fixed by patching Grafana on both test & prod (PR is not merged yet and there is no decision if they need JSON only or UI implementation).

Once (if?) Grafana PR is merged no patched installation will be needed.

For now I'm applying [this patch](https://github.com/cncf/devstats/blob/master/grafana/patches/teamplate_variable_value_optional_remove_from_url.patch) to grafana and generating install package myself. Then I'm installing it on test & prod instances (about 50 grafanas).

I've removed all template variables that contain postgres database variables - this fixes the original issue but many more hidden "under the hood" variables can be removed from URL that way.
It will make all URs prettier.

I'm doing this now.
